.gitattributes

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
The repo contains no files satisfying the templates given in this file. Is this file really needed?

Reply:
It's not harmful. What if someone comes up with a solution using zipped data resources for example and forget to create the file? Or what if someone adds a Word document to justify or explain the task?
The Git would better to know which files are binary. I usually save this file in all my projects.
If you don't like it, let's remove until it needed.